ext/configuration.h

```diff
@@ -126,6 +126,7 @@ enum ddtrace_sampling_rules_format {
     CONFIG(BOOL, DD_TRACE_ENABLED, "true", .ini_change = ddtrace_alter_dd_trace_disabled_config,               \
            .env_config_fallback = ddtrace_conf_otel_traces_exporter)                                           \
     CONFIG(BOOL, DD_INSTRUMENTATION_TELEMETRY_ENABLED, "true", .ini_change = zai_config_system_ini_change)     \
+    CONFIG(BOOL, DD_TELEMETRY_LOG_COLLECTION_ENABLED, "true", .ini_change = zai_config_system_ini_change)      \
```

Comment:
```suggestion
    CONFIG(BOOL, DD_TELEMETRY_LOG_COLLECTION_ENABLED, "true")      \
```
There's no need for having this system INI; just replace all checks with simple get_ instead of get_global_.